recipes/seq-seq-pan/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.0.0" %}
+
+package:
+    name: seq-seq-pan
+    version: {{ version }}
+
+source:
+    url: https://gitlab.com/chrjan/seq-seq-pan/repository/v{{ version }}/archive.tar.gz
+    md5: 2135730fee2eec0b018605bef4029f38
+    
+build:
+    number: 0
+    skip: True # [osx]
+    
+requirements:
+    run:
+        - python ==3.5 
```

Comment:
It requires py35 or higher so you should remove restriction here and instead include a skip for py27 in the build section.